homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
```

Comment:
What's the state of 
```py
    SensiboDeviceSensorEntityDescription(
        key="climate_react_type",
        translation_key="smart_type",
        value_fn=lambda data: data.smart_type,
        extra_fn=None,
        entity_registry_enabled_default=False,
    ),
```

Reply:
`temperature` , `feelslike` or `humidity`

Reply:
Should it be an enum sensor?

Reply:
Could be, I don't think we had that when this was created.